### PR TITLE
DRILL-7075: Fix debian package issue with control files

### PR DESCRIPTION
**Pull request by nareshgbhat**
Add control files to generate debian package.

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

### CODE REVIEW

File: distribution/src/deb/control/control [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Package: drill
+Version: 1.0-SNAPSHOT
```
Comment (vdiravka) [inline distribution/src/deb/control/control]: Please update Drill version in accoradance to the version from root POM file

File: distribution/src/deb/control/control [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+Section: misc
+Priority: optional
+Architecture: all
+Maintainer: name <>
+Description: Apache Drill
+Distribution: development
```
Comment (vdiravka) [inline distribution/src/deb/control/control]:
```suggestion
Distribution: development

```

Comment (vdiravka) [inline distribution/src/deb/control/control]: Please add: 
```
Homepage: http://drill.apache.org/
Description: Drill is an Apache open-source SQL query engine for Big Data exploration
```

**Comment by nareshgbhat** [comment]
@vdiravka Sure, I will update.  I have removed following line in distribution/pom.xml
<exclude>**/control</exclude> I can see it is still there in the root directory of pom.xml under drill.  I have tried to build by removing control variable all the places.  But I am still facing the build issues.  Hence I have created jira issue to add control file. Can you please give me the exact location where I have not removed "<exclude>**/control</exclude>" ?

**Comment by nareshgbhat** [comment]
@vdiravka AFAIK, my packaging knowledge says, we need control files either in static form or dynamic form.

**Comment by vdiravka** [comment]
Refer to your previous PR, which is actually merged to master branch:
https://github.com/apache/drill/commit/5e5b9a415d40899987bbc8fb7778e41779f599ad
`<exclude>**/control</exclude>`
Without this line `mvn clean install -DskipTests -Drat.skip=false -Dlicense.skip=false` will fail for your branch.

**Comment by nareshgbhat** [comment]
@vdiravka Thank you very much.  Let me know once it get merged.  I will close the PR and jira issue.

**Comment by vdiravka** [comment]
> @vdiravka Thank you very much. Let me know once it get merged. I will close the PR and jira issue.

No need to do that. Batch-commiter will merge PR and will close the jira. Usually batch commits are performed in the end of week. 
Thanks for making changes @nareshgbhat

**Pull request closed without merging**